docs/source/conf.py

```diff
@@ -54,6 +54,7 @@
     "collapse_navigation": False,
     "display_version": True,
     "logo_only": False,
+    "analytics_id": "G-Y4FB265L3N",
```

Comment:
where is it used ?

Reply:
Should be used by the theme here https://github.com/asteroid-team/asteroid_sphinx_theme/blob/master/asteroid_sphinx_theme/theme.conf#L8 but seems not to be working.

Reply:
I'll remove it.